core/src/main/java/org/stellar/anchor/auth/JwtService.java

```diff
@@ -193,7 +185,25 @@ public <T extends AbstractJwt> T decode(String cipher, Class<T> cls)
     }
   }
 
-  private String toBase64OrNull(String value) {
-    return value == null ? null : Base64.encodeBase64String(value.getBytes(StandardCharsets.UTF_8));
+  private SecretKey toSecretKeySpecOrNull(String secret) {
+    return isEmpty(secret) ? null : Keys.hmacShaKeyFor(((secret.getBytes(StandardCharsets.UTF_8))));
+  }
+
+  @SneakyThrows
+  public static Jws<Claims> getHeaderJwt(String signingKey, String cipher) {
+    var factory = KeyFactory.getInstance("Ed25519");
```

Comment:
Are we switching to asym key?

Reply:
asym key is used for a new feature (auth header signing with Stellar private keys, which are Ed25519), I'm drafting protocol change in parallel. Wanted to write some code first to make sure it's a working solution (and testing that it can be implemented in Java, JS, Ruby and Go -> primary langs where this feature)